sdk/keyvault/Azure.Security.KeyVault.Keys/tests/KeyVaultOnlyAttribute.cs

```diff
@@ -23,10 +23,14 @@ public TestCommand Wrap(TestCommand command)
             {
                 t.RunState = RunState.Skipped;
                 test.Properties.Set(PropertyNames.SkipReason, $"This test can only run on Key Vault.");
+                // Timeout cannot be set on SkipCommands
```

Comment:
Why is this even necessary, though? The whole test would be skipped, so maybe just don't set the timeout at all?

Reply:
Seems to be an issue with how NUnit implements the TimeoutCommand. It throws without this change here: 
https://github.com/nunit/nunit/blob/f0e35aa13e2e87fe9ad641115385afccab4bb5b5/src/NUnitFramework/framework/Internal/Commands/TimeoutCommand.cs#L38